workspaces/src/network/server.rs

```
@@ -39,6 +44,17 @@ impl SandboxServer {
     pub fn rpc_addr(&self) -> String {
         format!("http://localhost:{}", self.rpc_port)
     }
+
+    /// Wait until the RPC is available for sandbox. Will error out if this exceeds the amount of retries.
+    pub(crate) async fn wait_for_rpc(client: &Client) -> anyhow::Result<()> {
+        let retry_six_times = std::iter::repeat_with(|| Duration::from_millis(500)).take(6);
+        Retry::spawn(retry_six_times, || async { client.status().await })
```

Will this silently eat `JsonRpcError`s? Maybe we should log them?